sdcm/sct_events/group_common_events.py

```diff
@@ -474,6 +475,17 @@ def ignore_take_snapshot_failing():
         yield
 
 
+@contextmanager
+def ignore_coredump_on_network_block_disruption():
+    with ExitStack() as stack:
+        stack.enter_context(EventsSeverityChangerFilter(
+            new_severity=Severity.WARNING,
+            event_class=CoreDumpEvent,
+            regex=r".*executable=.*networkd.*executable_version=.*255.*",
```

Comment:
that's a bit too exact version, i.e. the bug might be on other versions as well

Reply:
I thought we can do `systemd version > 256` in code, regex doesn't fit such log very well

Reply:
now it can be removed

Reply:
Removed that